data/rewardTx/rewardTx.go

```diff
@@ -4,13 +4,28 @@ import (
 	"math/big"
 )
 
+const (
+	// LeaderTx identifies a leader reward tx type
+	LeaderTx = iota + 1
```

Comment:
I would change this line to:
```GO
LeaderTx byte = iota + 1
```
This will make all consts of type byte (default they are ints) and the runtime will not make useless casts from int to byte when creating reward transactions.

Reply:
But I used uint8 and not byte for the new filed added "type" in rewardTx

Reply:
Done